lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -47,6 +47,10 @@ class CriticalRequestChains extends ComputedArtifact {
       return false;
     }
 
+    if (request.statusCode === 0) {
+      return true;
```

Comment:
treat blocked request as critical
currently, in a normal run, blocked resources must be critical in the previous run (otherwise it's not possible to block it via the UI). This prevents some resources which start with low priority but end up with high priority become non-critical after being blocked. So user will be able to see and configure them in current UI.
This is a temp solution and will probably be removed after I implemented the bottom bar.